cinema/views.py

```diff
@@ -116,14 +148,22 @@ def get_serializer_class(self):
 class OrderPagination(PageNumberPagination):
     page_size = 10
     max_page_size = 100
+    permission_classes = (IsAuthenticated,)
```

Comment:
The `permission_classes` attribute is misplaced in the `OrderPagination` class. Pagination classes typically do not handle permissions. Consider removing this line or placing it in the appropriate viewset class.